test.py

```diff
@@ -0,0 +1,73 @@
+import logging
+import xml.etree.ElementTree as ET
+
+from panos.firewall import Firewall
+from panos.ha import HA1, HA2, HighAvailability
+from panos.network import EthernetInterface
+
+
+def main():
+    logging.basicConfig(level=0)
+
+    primary_fw = Firewall("10.8.202.107", "admin", "paloalto")
```

Comment:
Accidental commit (on this whole file)...?  If not, should be in the live tests directory or something..?

Reply:
whoops! Thanks for catching!